base/applications/network/netstat/lang/en-US.rc

```diff
@@ -27,76 +30,76 @@ END
 STRINGTABLE
 BEGIN
     IDS_IP4_STAT_HEADER "\nIPv4 Statistics\n\n"
-    IDS_IP_PACK_REC            "  Packets Received                  = %lu\n"
-    IDS_IP_HEAD_REC_ERROR      "  Received Header Errors            = %lu\n"
-    IDS_IP_ADDR_REC_ERROR      "  Received Address Errors           = %lu\n"
-    IDS_IP_DATAG_FWD           "  Datagrams Forwarded               = %lu\n"
-    IDS_IP_UNKNOWN_PRO_REC     "  Unknown Protocols Received        = %lu\n"
-    IDS_IP_REC_PACK_DISCARD    "  Received Packets Discarded        = %lu\n"
-    IDS_IP_REC_PACK_DELIVER    "  Received Packets Delivered        = %lu\n"
-    IDS_IP_OUT_REQUEST         "  Output Requests                   = %lu\n"
-    IDS_IP_ROUTE_DISCARD       "  Routing Discards                  = %lu\n"
-    IDS_IP_DISCARD_OUT_PACK    "  Discarded Output Packets          = %lu\n"
-    IDS_IP_OUT_PACKET_NO_ROUTE "  Output Packets No Route           = %lu\n"
-    IDS_IP_REASSEMBLE_REQUIRED "  Reassembly Required               = %lu\n"
-    IDS_IP_REASSEMBLE_SUCCESS  "  Reassembly Succesful              = %lu\n"
-    IDS_IP_REASSEMBLE_FAILURE  "  Reassembly Failures               = %lu\n"
-    IDS_IP_DATAG_FRAG_SUCCESS  "  Datagrams Successfully Fragmented = %lu\n"
-    IDS_IP_DATAG_FRAG_FAILURE  "  Datagrams Failing Fragmentation   = %lu\n"
-    IDS_IP_DATAG_FRAG_CREATE   "  Fragments Created                 = %lu\n"
+    IDS_IP_PACK_REC            "  Packets Received                   = %lu\n"
+    IDS_IP_HEAD_REC_ERROR      "  Received Header Errors             = %lu\n"
+    IDS_IP_ADDR_REC_ERROR      "  Received Address Errors            = %lu\n"
+    IDS_IP_DATAG_FWD           "  Datagrams Forwarded                = %lu\n"
+    IDS_IP_UNKNOWN_PRO_REC     "  Unknown Protocols Received         = %lu\n"
+    IDS_IP_REC_PACK_DISCARD    "  Received Packets Discarded         = %lu\n"
+    IDS_IP_REC_PACK_DELIVER    "  Received Packets Delivered         = %lu\n"
+    IDS_IP_OUT_REQUEST         "  Output Requests                    = %lu\n"
+    IDS_IP_ROUTE_DISCARD       "  Routing Discards                   = %lu\n"
+    IDS_IP_DISCARD_OUT_PACK    "  Discarded Output Packets           = %lu\n"
+    IDS_IP_OUT_PACKET_NO_ROUTE "  Output Packet No Route             = %lu\n"
+    IDS_IP_REASSEMBLE_REQUIRED "  Reassembly Required                = %lu\n"
+    IDS_IP_REASSEMBLE_SUCCESS  "  Reassembly Successful              = %lu\n"
+    IDS_IP_REASSEMBLE_FAILURE  "  Reassembly Failures                = %lu\n"
+    IDS_IP_DATAG_FRAG_SUCCESS  "  Datagrams Successfully Fragmented  = %lu\n"
+    IDS_IP_DATAG_FRAG_FAILURE  "  Datagrams Failing Fragmentation    = %lu\n"
+    IDS_IP_DATAG_FRAG_CREATE   "  Fragments Created                  = %lu\n"
```

Comment:
Why the extra space in all lines?

Reply:
As I already wrote, that matches XP output, and was changed without a word in a recent PR...

Reply:
Is the aligning whitespace size the same in other localizations?

Reply:
I have no idea: ROS has EN-US only, I compared to EN-US WXP (only)...